src/components/game/Board.js

```diff
@@ -6,7 +6,7 @@ import { connect } from 'react-redux'
 
 export class Board extends PureComponent {
   static propTypes = {
-    board1: PropTypes.arrayOf(
+    board: PropTypes.arrayOf(
```

Comment:
this will be a different board for both players, right? am getting confused because we named it board1 but it's been a while so idk anymore but probably it wasn't necessary then haha

Reply:
Oh don't worry! I just changed the name of the boards to be the same as the ones in the redux state :smile:.
Yes the boards will be different for each players